### PR TITLE
Neutron Public IPv4 usage billing

### PR DESCRIPTION
**Pull request by xroot88**
A CUF notification for neutron public IPv4 usage.

### CODE REVIEW

**Comment by stajkowski** [comment]
Dragon, possible we can get this reviewed?

File: yagi/handler/notification.py [hunk 1]
```diff
@@ -24,6 +26,34 @@
...
+"""id="{id}" """
+"""version="1" """
+"""resourceId="{resourceId}" """
+"""resourceName="{resourceName}" """
+"""tenantId="{tenantId}" """
+"""startTime="{startTime}" """
+"""endTime="{endTime}" """
+"""type="USAGE" """
+"""dataCenter="{dataCenter}" """
+"""region="{region}" """
```
Comment (DragonDM) [inline yagi/handler/notification.py]: Missing a closing >

File: yagi/handler/notification.py [hunk 2]
```diff
@@ -24,6 +26,34 @@
...
+"""<neutron:product serviceCode="CloudNetworks" """
+"""resourceType="{resourceType}" """
+"""ipType="{ipType}"/> """
+"""</event>"""
+)
+
+atom_hopper_time_format = "%Y-%m-%dT%H:%M:%SZ"
```
Comment (DragonDM) [inline yagi/handler/notification.py]: Duplicated with line 57 below

File: yagi/handler/notification.py [hunk 3]
```diff
@@ -24,6 +26,34 @@
...
+
+LOG = logging.getLogger(__name__)
```
Comment (DragonDM) [inline yagi/handler/notification.py]: dup of line 59

File: yagi/handler/notification.py [hunk 4]
```diff
@@ -24,6 +26,34 @@
...
+
+
+class BaseNotification(object):
```
Comment (DragonDM) [inline yagi/handler/notification.py]: Part of this class is duplicated. Bad merge?

**Pull request merged**